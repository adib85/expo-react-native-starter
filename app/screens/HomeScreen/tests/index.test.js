/**
 * Test the HomePage
 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import { HomeScreen } from '../index';

it('renders correctly', () => {
  const tree = renderer
    .create(<HomeScreen loading error={false} repos={[]} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
