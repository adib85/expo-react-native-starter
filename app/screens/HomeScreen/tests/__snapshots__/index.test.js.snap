// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "marginTop": 50,
        },
        undefined,
      ]
    }
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
      >
        Username
      </Text>
      <TextInput
        id="username"
        onChangeText={undefined}
        placeholder="mxstbr"
        style={
          Array [
            Object {},
            undefined,
          ]
        }
        type="text"
        value={undefined}
      />
    </View>
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Get repositories"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#841584",
              },
            ]
          }
        >
          Load
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <RCTScrollView>
      <View />
    </RCTScrollView>
  </View>
</View>
`;
